src/tests/runTests.ts

```diff
@@ -1,22 +1,26 @@
 import * as path from 'path';
 import { runTests } from '@vscode/test-electron';
 
- async function main() {
- 	try {
- 		// The folder containing the Extension Manifest package.json
- 		// Passed to `--extensionDevelopmentPath`
- 		const extensionDevelopmentPath = path.resolve(__dirname, '../../');
+async function main() {
+    try {
+        // The folder containing the Extension Manifest package.json
+        // Passed to `--extensionDevelopmentPath`
+        const extensionDevelopmentPath = path.resolve(__dirname, '../../');
 
- 		// The path to test runner
- 		// Passed to --extensionTestsPath
- 		const extensionTestsPath = path.resolve(__dirname, './suite/index');
+        // The path to test runner
+        // Passed to --extensionTestsPath
+        const extensionTestsPath = path.resolve(__dirname, './suite/index');
 
- 		// Download VS Code, unzip it and run the integration test
- 		await runTests({ extensionDevelopmentPath, extensionTestsPath });
- 	} catch (err) {
- 		console.error('Failed to run tests');
- 		process.exit(1);
- 	}
- }
+        // Download VS Code, unzip it and run the integration test
+        await runTests({ extensionDevelopmentPath, extensionTestsPath });
+    } catch (err) {
+        console.error(`Failed to run tests:\n${err}`);
```

Comment:
Added this extra logging because swallowing the error here caused hours of head-scratching today.